src/components/ProjectListItemConnected/ProjectListItemConnected.tsx

```diff
@@ -32,18 +35,40 @@ export const ProjectListItemConnected: FC<ProjectListItemConnectedProps> = ({
     parent,
     project,
     firstLevel,
+    subTree,
+    mainProject,
     ...props
 }) => {
-    const { view, queryState } = useUrlFilterParams({
+    const { view, hideEmptyProjects } = useUrlFilterParams({
         preset: filterPreset,
     });
+    const [isProjectEmpty, setIsProjectEmpty] = useState(() => {
+        if (subTree) {
```

Comment:
move this function inside separately helper, cuz you make two identical functions
the same function have below